### PR TITLE
[ACA-2078] fix Flickering viewer toolbar icons when navigating to next/previous …

### PR DESCRIPTION
**Pull request by eromano**
…files

**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [X] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
fix Flickering viewer toolbar

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/app/components/viewer/viewer.component.ts [hunk 1]
```diff
@@ -131,8 +131,13 @@ export class AppViewerComponent implements OnInit, OnDestroy {
       .subscribe((ruleContext) => {
         this.selection = ruleContext.selection;
 
-        this.toolbarActions = this.extensions.getViewerToolbarActions();
-        this.openWith = this.extensions.openWithActions;
+        if (this.toolbarActions.length === 0) {
+          this.toolbarActions = this.extensions.getViewerToolbarActions();
```
Comment (DenysVuika) [inline src/app/components/viewer/viewer.component.ts:135]: this may prevent evaluation of visibility rules

Comment (eromano) [reply src/app/components/viewer/viewer.component.ts:135]: is to avoid reload it after the first time, can they change after the first load?

Comment (DenysVuika) [reply src/app/components/viewer/viewer.component.ts:135]: the `getViewerToolbarActions()` checks visibility conditions each time, maybe if you select folder of file, the buttons may change

Comment (DenysVuika) [reply src/app/components/viewer/viewer.component.ts:135]: anyway, might need some checks if nothing gets brocken

Comment (eromano) [reply src/app/components/viewer/viewer.component.ts:135]: I did some checks didn't find a problem for now

**Pull request merged**